Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -147,22 +167,35 @@ updateSeats()
               ])
           ],
           [
-            "You can add additional team members at anytime from your account page."
+            """
+You must have at least two seats for your team subscription. You can add additional team members at anytime
+from your account page.
+"""
           ]
         )
       ]
     )
   ]
 }
 
-private func teamOwner(_ currentUser: User) -> Node {
+private func teamOwner(currentUser: User, subscribeData: SubscribeConfirmationData) -> Node {
+  guard subscribeData.isOwnerTakingSeat else {
+    return input([
+      name(SubscribeData.CodingKeys.isOwnerTakingSeat.rawValue),
+      `type`(.hidden),
+      value("false")
+      ])
+  }
+
   return gridColumn(
     sizes: [.mobile: 12],
     [
+      id("team-owner"),
       `class`([
         Class.border.all,
         Class.pf.colors.border.gray850,
-        Class.padding([.mobile: [.all: 2]])
+        Class.padding([.mobile: [.all: 2]]),
+        Class.margin([.mobile: [.top: 1]]),
```

Comment:
Added this to match the padding that the team cells have. Makes clicking "remove self" a bit more of a seamless transition. (Did not regenerate snaps...but can if no more changes are coming.)